src/ui/units/dash/containers/Dialogs/DialogRelations/DialogRelations.tsx

```diff
@@ -61,16 +63,23 @@ const DialogRelations = (props: DialogRelationsProps) => {
     const showDebugInfo = useSelector(selectDebugMode);
     const dashTabAliases = useSelector(getCurrentTabAliases);
 
+    const aliasWarnButtonRef = React.useRef<HTMLElement | null>(null);
+
+    const [aliasWarnPopupOpen, setAliasWarnPopupOpen] = React.useState(false);
     const [searchValue, setSearchValue] = React.useState('');
     const [typeValues, setTypeValues] = React.useState(DEFAULT_FILTERS);
     const [changedWidgets, setChangedWidgets] = React.useState<WidgetsTypes>();
     const [preparedRelations, setPreparedRelations] = React.useState<DashMetaData>([]);
     const [aliases, setAliases] = React.useState(dashTabAliases || {});
 
-    const {isLoading, currentWidgetMeta, relations, datasets, dashWidgetsMeta} = useRelations({
-        dashKitRef,
-        widget,
-    });
+    const {isLoading, currentWidgetMeta, relations, datasets, dashWidgetsMeta, invalidAliases} =
+        useRelations({
+            dashKitRef,
+            widget,
+            dialogAliases: aliases,
+        });
+
+    const [showInvalidAliases, setShowInvalidAliases] = React.useState<string[] | null>(null);
```

Comment:
`showInvalidAliases` sounds like a boolean type. Perhaps it would be better to rename?

Reply:
renamed